app/css/icons/tag.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="30" width="30" version="1.1"><g transform="matrix(.019697 0 0 -.019697 .079568 27.813)"><path fill="#fff" d="m448 1088q0 53-37.5 90.5t-90.5 37.5-90.5-37.5-37.5-90.5 37.5-90.5 90.5-37.5 90.5 37.5 37.5 90.5zm1067-576q0-53-37-90l-491-492q-39-37-91-37-53 0-90 37l-715 716q-38 37-64.5 101t-26.5 117v416q0 52 38 90t90 38h416q53 0 117-26.5t102-64.5l715-714q37-39 37-91z"/></g></svg>
```

Comment:
This file can be minified further. I usually only use a `viewBox` attribute in lieu of `height` and `width`. The `version` attribute can be removed to.
It's better if the `transform` tag is removed and its effect merged into the `path` values.

I can do that operation for you if you want.